cdap-unit-test/src/test/java/io/cdap/cdap/security/AuthorizationTest.java

```diff
@@ -782,8 +785,18 @@ private void testCrossNSDatasetAccessWithAuthMapReduce(MapReduceManager mrManage
 
     Map<EntityId, Set<? extends Permission>> neededPrivileges =
       ImmutableMap.<EntityId, Set<? extends Permission>>builder()
-      .put(inputDatasetNSId, EnumSet.allOf(StandardPermission.class))
-      .put(outputDatasetNSId, EnumSet.allOf(StandardPermission.class))
+      .put(inputDatasetNSId, new HashSet<Permission>() {
```

Comment:
nit: kind of odd to extend HashSet like this, would be more readable to create the HashSet normally and call addAll() on it, then re-use it everywhere.

Reply:
updated to re-use the same set of namespace admin permissions